pyproject.toml

```diff
@@ -52,8 +52,8 @@ dependencies = [
 # should be OK.
 test = [
     "cython-test-exception-raiser >= 1.0.2, <2",
-    "PyHamcrest >= 1.9.0",
-    "hypothesis ~= 6.0",
+    "PyHamcrest >= 2",
```

Comment:
For testing I went with minimum that are quite new.
I think this should be ok as we want to run the tests in a "modern" environment.